target/linux/qualcommax/ipq60xx/base-files/etc/uci-defaults/01_promisc_mode

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+yuncore,fap650)
+	uci set network.wan_dev=device
+	uci set network.wan_dev.name='wan'
+	uci set network.wan_dev.promisc='1'
```

Comment:
What is this script supposed to do?

Can you please stop adding stuff like this without any explanation

Reply:
Well, this is not the solution for this at all, I find it hard to believe that this is the only device that has this issue if there is one

Reply:
I've rabased my PR branch to master and problem is solved. So this hack is removed.